### DISCUSSION TITLE
Question that's probably asked alot but important to me:

### DISCUSSION THREAD

**Discussion by aguywhosaguy** [Tyrrrz/DiscordChatExporter#953] [Q&A]
I've always been protective of my discord account. However, I want to backup what matters to me the most, which are my discord messages. My only problem is that I have quite a lot of messages having been on the platform for 4 years. Is there a chance that using the program will 100% get me banned? Or is there a really low chance?

**Comment by voidf1sh** [comment]
Automating a user account is against TOS and there is the risk the Discord will detect this and suspend or delete your account permanently. Not sure how high the odds are but I'm not risking it with my main account more often than necessary.
